### ISSUE TITLE
Issues refreshing pickerItems 

### ISSUE THREAD

**Issue by manuelvanrijn** [ra1028/Former#6]
Hi,

Maybe I'm missing something but I'm having some troubles refreshing the `pickerItems` on a `InlinePickerRowFormer`

My goal is to load different items in a `InlinePickerRowFormer` based on the selection of another `InlinePickerRowFormer`

I've tried removing and adding items calling `update()` on the row and `self.somerow.former?.reload(rowFormer: self.otherrow)` but both seem to ignore the capacity and aren't showing the option correctly.

So basically I'm wondering what the correct way is, for refreshing the `pickerItems` on a `InlinePickerRowFormer`

**Comment by jmscarpa** [comment]
@manuelvanrijn any luck with this issue?

**Comment by manuelvanrijn** [comment]
@jmscarpa nop, I've resolved it using a PushRow instead. Although now I'm looking at the source again and maybe triggering `evaluateDisabled()` on the target row might work, after setting the new options.
